test/vfd_swmr_group_writer.c

```diff
@@ -112,216 +157,2736 @@ state_init(state_t *s, int argc, char **argv)
                     goto error;
                 }
 
-                if (ch == 'a')
-                    s->asteps = (unsigned)tmp;
-                else if (ch == 'c')
-                    s->csteps = (unsigned)tmp;
-                else if (ch == 'n')
-                    s->nsteps = (unsigned)tmp;
-                else if (ch == 'u')
-                    s->update_interval = (unsigned)tmp;
-                break;
-            case 'b':
-                s->filetype = H5T_STD_U32BE;
-                break;
-            case 'N':
-                s->use_named_pipes = false;
-                break;
-            case 'q':
-                verbosity = 0;
-                break;
-            case '?':
-            default:
-                usage(s->progname);
-                break;
+                if (ch == 'a')
+                    s->asteps = (unsigned)tmp;
+                else if (ch == 'c')
+                    s->csteps = (unsigned)tmp;
+                else if (ch == 'n')
+                    s->nsteps = (unsigned)tmp;
+                else if (ch == 'u')
+                    s->update_interval = (unsigned)tmp;
+                break;
+            case 'b':
+                s->filetype = H5T_STD_U32BE;
+                break;
+            case 'N':
+                s->use_named_pipes = false;
+                break;
+            case 'A':
+                if (strcmp(optarg, "compact") == 0)
```

Comment:
These string operations should have the prefix `HD`.  So do other string functions in this test.

Reply:
will do.